src/main/java/frc/robot/subsystems/drive/DrivetrainStateMachine.java

```diff
@@ -10,14 +10,24 @@
 
 public class DrivetrainStateMachine extends StateMachine {
 
-    public TeleopState teleopSwerve = new TeleopState();
-    public FollowTag aprilTagState = new FollowTag();
+    public static BalanceState balanceState = new BalanceState();
+    public static FollowTag aprilTag = new FollowTag();
+    public static ScoreState scoreState = new ScoreState();
+    public static TeleopState teleopState = new TeleopState();
 
     private static XboxController driverController = RobotMap.driverController;
 
     public DrivetrainStateMachine() {
+        
+        aprilTag.build();
+        balanceState.build();
+        scoreState.build();
+        teleopState.build();
+        
 
-        Supplier<Boolean> checkAButton = () -> {
+
+
+       /*  Supplier<Boolean> checkAButton = () -> {
```

Comment:
Remove Code that we do not use anymore, do not just comment it out. We have githug for looking at file history. No need to keep a comment record. It just makes the code less clean.